appveyor.yml

```diff
@@ -41,6 +43,12 @@ environment:
         NUMPY_VERSION: "1.10"
         ASTROPY_VERSION: "lts"
 
+      - PYTHON_VERSION: "3.5"
+        NUMPY_VERSION: "stable"
```

Comment:
The run time is dominated by far by the development builds, so I don't think we'll gain much by removing these two, but you're right that they aren't technically needed.